README.md

```diff
@@ -21,8 +21,34 @@ Script to interact with Redbrick LDAP.
 go get github.com/redbrick/rbldap/cmd/rb-ldap
 ```
 
+#### Installation with Docker
+
+There is a `docker-compose.yml` for rb-ldap. Before running this file, you should make sure to change the paths to the ldap secrets, you can do this in the `docker-compose.yml` itself, under volumes.
+
+The docker-compose file can be run using:
+
+```console
+docker-compose run --rm rb-ldap
+```
+
+If you want to use it this way, we really recommend you create an alias to a function, it's much easier. A function such as the following can be placed in your `~/.bashrc`:
+
+```bash
+rb_ldap() {
+        pushd "/path/to/your/docker-compose/file/"
+        /path/to/docker-compose/ docker-compose run --rm rb-ldap "${@:1}"
+        popd
+}
+
+alias rb-ldap=rb_ldap
```

Comment:
why use an alias why not just call the function `rb-ldap`

Reply:
Honestly an oversight at the time. Will make the change.